EventFiltering/PWGCF/CFFilterTwoN.cxx

```diff
@@ -0,0 +1,360 @@
+enum kCFTwoBodyTriggers {
+  kPD, //=0
+  kLD, //=1
+  kLAST_CFTwoBodyTriggers
+};
+
+enum kDetector {
+  kTPC,
+  kTPCTOF,
+  kNdetectors,
+  kLAST_Detector
+};
+
+static const std::vector<std::string> CfTriggerNames{"kPD", "kLD"};
+static constexpr uint8_t Track = 0;
+static constexpr uint8_t V0 = 1; // V0
+// static constexpr uint8_t V0Daughter = 2; // V0  daughters
+
+static constexpr uint32_t kSignMinusMask = 1;
+static constexpr uint32_t kSignPlusMask = 2;
+
+// static constexpr uint32_t knSigmaProton = 48;
+static constexpr uint32_t kValue0 = 0;
+
+} // namespace
+
+namespace o2::aod
+{
+using FullCollision = soa::Join<aod::Collisions,
+                                aod::EvSels,
+                                aod::Mults>::iterator;
+} // namespace o2::aod
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+using namespace o2::analysis::femtoDream;
+
+struct CFFilterTwoN {
+
+  Produces<aod::CFFiltersTwoN> tags;
+
+  Configurable<float> confKstarTriggerLimit{"KstarTriggerLimitUpper", 1.4f, "Kstar limit for selection"};
+  Configurable<int> KstarTrigger{"KstarTrigger", 0, "Choice which trigger to run"};
+
+  Configurable<float> confNSigma{"nSigma", 3.5f, "nSigma for TPC/TPC&TOF selection"};
+  Configurable<float> confPIDThreshold{"PThreshold", 0.75f, "P threshold for TPC/TPC&TOF selection"};
+
+  Configurable<float> ldeltaPhiMax{"ldeltaPhiMax", 0.010, "Max limit of delta phi"};
+  Configurable<float> ldeltaEtaMax{"ldeltaEtaMax", 0.010, "Max limit of delta eta"};
+
+  // Obtain particle candidates of protons, deuterons as well as antiprotons and antideuterons
+  Partition<o2::aod::FemtoDreamParticles> partPD = (o2::aod::femtodreamparticle::partType == Track) &&
+                                                   ((o2::aod::femtodreamparticle::cut & kSignPlusMask) > kValue0);
+  Partition<o2::aod::FemtoDreamParticles> partAntiPD = (o2::aod::femtodreamparticle::partType == Track) &&
+                                                       ((o2::aod::femtodreamparticle::cut & kSignMinusMask) > kValue0);
+  Partition<o2::aod::FemtoDreamParticles> partL = (o2::aod::femtodreamparticle::partType == V0) &&
+                                                  ((o2::aod::femtodreamparticle::cut & kSignPlusMask) > kValue0);
+  Partition<o2::aod::FemtoDreamParticles> partAntiL = (o2::aod::femtodreamparticle::partType == V0) &&
+                                                      ((o2::aod::femtodreamparticle::cut & kSignMinusMask) > kValue0);
+
+  HistogramRegistry registry{"registry", {}, OutputObjHandlingPolicy::AnalysisObject};
+  HistogramRegistry registryQA{"registryQA", {}, OutputObjHandlingPolicy::AnalysisObject};
+
+  FemtoDreamDetaDphiStar<aod::femtodreamparticle::ParticleType::kTrack, aod::femtodreamparticle::ParticleType::kTrack> closePairRejectionTT;
+  // FemtoDreamDetaDphiStar<aod::femtodreamparticle::ParticleType::kTrack, aod::femtodreamparticle::ParticleType::kV0> closePairRejectionTV0;
```

Comment:
this is needed when you use lambdas, because you want to check the primary track vs secondary tracks of V0